src/main/java/codechicken/nei/ClientHandler.java

```diff
@@ -235,7 +235,6 @@ public static void load() {
 
     public static void postInit() {
         loadHandlerOrdering();
-        loadPluginsList();
```

Comment:
Please explain why you moved this method to a later loading stage.

Reply:
this is necessary to fix this problem: https://github.com/GTNewHorizons/NotEnoughItems/issues/524
Immersive Engineering breaks if you load plugins in postInit